MVC/config/config.php

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
Please keep the the following file inside `.gitignore` since it contains your database information